src/python/doc/installation.rst

```diff
@@ -98,6 +98,18 @@ following command in a terminal:
     export PYTHONPATH='$PYTHONPATH:/path-to-gudhi/build/python'
     make test
```

Comment:
Shouldn't we directly say to run `ctest` (possibly suggesting `-j8`, `--output-on-failure` as optional options)?

Reply:
Done on 38f0ff98cf6da1e882b6b44fc3ed7b3d310fb91f (+ conflict resolution cec28c5ee04886ff30d4f3960d713e044837ff4f )